src/components/Profile.jsx

```diff
@@ -16,5 +17,12 @@ class Profile extends React.Component {
 		);
 	}
 }
+Profile.propTypes = {
+	user: PropTypes.shape({
+		firstName: PropTypes.string.isRequired,
+		lastName: PropTypes.string.isRequired,
+		images: PropTypes.array.isRequired,
+	}),
```

Comment:
Shape looks great! The only thing I'd say is that the user _itself_ should also be required. You can do that by adding `.isRequired` on the end of the shape definition.